packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/return-info-popover.tsx

```diff
@@ -0,0 +1,70 @@
+import { InformationCircleSolid } from "@medusajs/icons"
+import { AdminReturn } from "@medusajs/types"
+import { Badge, Popover, Text } from "@medusajs/ui"
+import { useState } from "react"
+import { useTranslation } from "react-i18next"
+import { formatDate } from "../../../../../components/common/date"
+
+type ReturnInfoPopoverProps = {
+  orderReturn: AdminReturn
+}
+
+function ReturnInfoPopover({ orderReturn }: ReturnInfoPopoverProps) {
+  const { t } = useTranslation()
+  const [open, setOpen] = useState(false)
+
+  const handleMouseEnter = () => {
+    setOpen(true)
+  }
+
+  const handleMouseLeave = () => {
+    setOpen(false)
+  }
+
+  if (typeof orderReturn !== "object") {
+    return
+  }
+
+  return (
+    <Popover open={open}>
+      <Popover.Trigger
+        onMouseEnter={handleMouseEnter}
+        onMouseLeave={handleMouseLeave}
+        autoFocus={false}
+        className="focus-visible:outline-none align-sub"
+      >
+        <InformationCircleSolid />
+      </Popover.Trigger>
+
+      <Popover.Content
+        align="center"
+        side="top"
+        className="bg-ui-bg-component focus-visible:outline-none p-2"
+      >
+        <div className="flex flex-col">
+          <Badge size="2xsmall" className="mb-2" rounded="full">
+            #{orderReturn.id.replace("return_", "")}
```

Comment:
suggestion: can we keep the same number of characters consistent across the page so users can easily identifies the same return